aibrary/resources/aibrary_wrapper.py

```diff
@@ -0,0 +1,26 @@
+import os
+
+import openai
+
+from aibrary.resources.translation import TranslationClient
+
+
+class AiBrary(openai.OpenAI):
```

Comment:
Please add a method to get all models.
It would be good if the method also gets an optional field `category` and returns models in that category.